tinytuya/Contrib/ThermostatDevice.py

```diff
@@ -616,7 +616,8 @@ def __bytes__( self ):
                         coolto -= coolmod
                         if coolmod >= 25: coolto += 50
 
-                    log.info( 'CF is: %r %r %r cool: %r %r %r', cf, self.heatto, heatto / 100, self.coolto, coolto / 100, self.time )
+                    # FIXME: make a logger available here
+                    #log.info( 'CF is: %r %r %r cool: %r %r %r', cf, self.heatto, heatto / 100, self.coolto, coolto / 100, self.time )

```

Comment:
FIXME - I'm sure you know, just adding to my review

Reply:
Yeah, I'm not sure what I'm going to do about that one, it's buried like 3 classes deep.  I may just remove it, though having a logger available there for debugging may be needed at some point.

Reply:
Sortof gross, but possibly define a global scoped log (glog) to use for this one?